libc/unistd_getopt.c

```diff
@@ -314,6 +314,33 @@ TEST(unistd_getopt, getopt_unknownopt_noarg)
 }
 
 
+TEST(unistd_getopt, getopt_unknownopt_multiple)
+{
+	/* mocks of main() arguments */
+	char *testargv[] = { "cmd", "-xxx", "", "" };
```

Comment:
use two separate `const char *` arrays if needed for test

Reply:
I would leave it without `const` here, tried to change, but it becomes complicated then (getopt requires char * const argv[])